### ISSUE TITLE
Unclosed file `senter/model` warnings/errors.

### ISSUE THREAD

**Issue by Stannislav** [explosion/spaCy#8730]
I encountered this issue while using `pytest`. The issue is only visible after turning warnings into errors. 

## How to reproduce the behaviour
```python
# test_something.py
import pytest
import spacy


@pytest.fixture()
def nlp():
    return spacy.load("en_core_web_sm")


def test_something(nlp):
    pass
```

Run this with `pytest` with warnings turned into errors:
```bash
pytest -We -W "ignore::DeprecationWarning" test_something.py
```

Output:
```none
=============================================================== ERRORS ================================================================
_____________________________________________________ ERROR at setup of test_something ________________________________________________
...
ResourceWarning: unclosed file <_io.BufferedReader name='/some/path/venv/lib/python3.9/site-packages/en_core_web_sm/en_core_web_sm-3.0.0/senter/model'>
```

(the option `-W "ignore::DeprecationWarning"` is to get rid of another unrelated warning, maybe that one could be worth another ticket?)

## Your Environment

* Operating System: macOS, Big Sur, 11.4
* Python Version Used: `3.9.5`
* spaCy Version Used: `spacy==3.0.6`
* Environment Information: `pytest==6.2.4`

**Comment by adrianeboyd** [comment]
Hi, I think this was fixed in #8244. This fix should already be in v3.1.0 and we're planning a patch/backport release as v3.0.7 that should include this in the future.

**Comment by Stannislav** [comment]
Awesome, thanks!
